src/main/fc/fc_hardfaults.c

```diff
@@ -116,10 +116,8 @@ void HardFault_Handler(void)
     LED0_OFF;
 
     while (1) {
-#ifdef LED2
```

Comment:
There's no `#define LED2` anywhere, beside first line of this function accesses LED2 without any check.